frescobaldi_app/scorewiz/build.py

```diff
@@ -132,6 +132,10 @@ def assignMusic(self, name=None, octave=0, transposition=None):
             ly.dom.Pitch(toct, tnote, fractions.Fraction(talter, 2), ly.dom.Transposition(s))
         ly.dom.LineComment(_("Music follows here."), s)
         ly.dom.BlankLine(s)
+        num = self.scoreProperties.blankStaff.value()
+        if num:
+            ly.dom.Text(r"\repeat unfold {num} {{ s1 \break }}".format(num=num), s)
```

Comment:
I'm not proud of this part... If there is a better way to do it, please enlighten me !

Reply:
I took the time signature and `\partial` into account. I did not find anything in the dom code about repeats. Nothing about rests either.